lib/ibm/cloud/sdk/version.rb

```diff
@@ -1,7 +1,7 @@
 module IBM
   module Cloud
     module SDK
-      VERSION = "0.1.2"
+      VERSION = "0.1.3"
```

Comment:
@Kuldip-Nanda the gem version shouldn't be bumped in a PR but when the gem is released by a gem owner.  The reason being the owner might want to group multiple changes together before releasing and also it is up to the owner to say what is a "breaking change"